content/quiz_meets.md

```diff
@@ -1,6 +1,13 @@
 # Quiz Meets
+The tournament brackets are based on three things:   A preliminary round, an elimination round (at the discretion of the meet director), and championship quizzes.
 
 ## Preliminary Rounds
+The teams will all receive team points as follows:
+- First place in a quiz:  10 points, plus 1 point for every 10 points over 100
+- Second place in a quiz:  5 points, plus 1 point for every 10 points over 60
+- Third place in a quiz:  1 point, plus 1 point for every 10 points over 30
+In case of a tie, points are awarded according to the team score at the end of question 20.
+If ties are not being broken in prelims, more than one team can receive First place, Second place, or Third place points.
```

Comment:
Added by me.

Reply:
I'm thinking this is duplicated in the English-Script somewhere. Should this be part of scoring?

Reply:
It's actually duplicated in this doc. Under Team Points Calculation.

If I remove the content under Preliminary Rounds, that's all that needs to happen right?

Reply:
You could remove the non-ES portion, yes. As long as you think the overall doc flow is still good.

Reply:
I did, but kept a non-ES partial description.